xlators/mgmt/glusterd/src/glusterd-mountbroker.c

```diff
@@ -260,10 +260,11 @@ make_georep_mountspec(gf_mount_spec_t *mspec, const char *volnames, char *user,
     int vc = 0;
     int i = 0;
     int ret = 0;
+    xlator_t *this = THIS;
```

Comment:
I don't thing we need again new var and assignment to THIS. why cant we use THIS in gf_smsg?

Reply:
Technically we are creating one more variable for this. The idea was to not to have THIS inside function calls.

Reply:
Seeing the overall consensus, I think your suggestion was prudent. Will go with THIS in here..

Reply:
Yes, I believe @Sheetalpamecha is right - it's mostly because we only call it once, and we only call it once in error paths - so I think it's the right direction.